visRayTracer.mac

```diff
@@ -0,0 +1,32 @@
+# Visualization and setup macro 
+# This file is run by default if you just type WCSim and nothing else.
```

Comment:
Replace the comments on lines 1 and 2 with relevant ones for visRayTracer.mac. This one isn't run by default in WCSim. Put something like, "This macro is used to visualize geometries using Raytracer. The output of this macro is three .jpeg files visualizing the geometry from three different viewpoints."